kornia/enhance/adjust.py

```diff
@@ -4,6 +4,7 @@
 import torch.nn as nn
 import torch.nn.functional as F
 
+from kornia.augmentation.utils import _transform_input3d
```

Comment:
we shouldn't use `konira.augmentation` in the library core. In theory `kornia.augmentation` should be a standalone package that depends on kornia, but not viceversa. `_transform_input3d` must moved elsewhere in the library, maybe under utilites but not expose in the high level. /cc @shijianjian

Reply:
Maybe I could move it to `kornia.utils.image`, the location of `_to_bchw`. But this would be a breaking change, that could cause problems with projects that depend on kornia. 

My suggestion is to copy the code of `_transform_input3d` to `kornia.utils.image`. And I can add a deprecation warning to `kornia.augmentation.utils._transform_input3d`, saying it will be removed in the future versions.

Reply:
`_to_bcdhw` to `kornia.utils.image` sounds good to me